virttest/shared/cfg/guest-hw.cfg

```diff
@@ -23,6 +23,10 @@ variants:
         only q35
         required_qemu = [2.6.0, )
         nic_model = e1000e
+    - igb:
+        only q35
+        required_qemu = [8.1.0, )
```

Comment:
igb device was introduced from qemu-8.0.0. https://www.mail-archive.com/qemu-devel@nongnu.org/msg947415.html

Reply:
> igb device was introduced from qemu-8.0.0. https://www.mail-archive.com/qemu-devel@nongnu.org/msg947415.html

Downstream qemu 8.0.0 only included a part of code. According to the internal bug it is supported starting from 8.1.0-4 .

Reply:
This is an upstream project and should not only be considered downstream.

Reply:
> This is an upstream project and should not only be considered downstream.

Changed to qemu-8.0.0, please help review it again.